packaging/nfpm/mimir/preremove.sh

```diff
@@ -13,13 +13,21 @@ uninstall() {
     systemctl stop mimir.service ||:
 }
 
+upgrade() {
+    :
+    # printf "\033[32m Post Install of an upgrade\033[0m\n"
```

Comment:
Can we just remove this comment? I'm not sure why there's commented out code there.